plugins/providers/metabase/provider.go

```diff
@@ -45,14 +45,14 @@ func (p *provider) GetResources(pc *domain.ProviderConfig) ([]*domain.Resource,
 		return nil, err
 	}
 
-	var resourceTypes []string
+	var resourceTypes = make(map[string]bool, 0)
 	for _, rc := range pc.Resources {
-		resourceTypes = append(resourceTypes, rc.Type)
+		resourceTypes[rc.Type] = true
 	}
 
 	resources := []*domain.Resource{}
 
-	if containsString(resourceTypes, ResourceTypeDatabase) {
+	if _, ok := resourceTypes[ResourceTypeDatabase]; ok && resourceTypes[ResourceTypeDatabase] {
```

Comment:
wouldn't this value `resourceTypes[ResourceTypeDatabase]` be true always if found in map at this line?

Reply:
I am checking this to confirm the value in the map is true.